### PR TITLE
visual message if archive is not found on clicking recent or first

### PR DESCRIPTION
**Pull request by rgpt**
This resolves [issue#39](https://github.com/internetarchive/wayback-machine-chrome/issues/39) 
It is not only the display message that we need to change if you check the code the two scripts background.js and pop.js are taking to each other using message passing 

and recent capture , first capture is regulated in same way. There was no proper response. 
So, i am sending response based on whether url is found in archive or not.

based on that i am notifying using chrome Notifications.

even if we do not want the user to get notified by chrome notification and inside the extension itself still the code to send the response back will work the same. We can change it as required

**previous:**

After sending response and handling from the pop.js where it is supposed to be handled.
**Now:**

### CODE REVIEW

File: scripts/popup.js [hunk 1]
```diff
@@ -4,17 +4,45 @@ function save_now_function(){
...
 
 function first_capture_function(){
 	var wb_url = "https://web.archive.org/web/0/";
 	chrome.runtime.sendMessage({message: "openurl", wayback_url: wb_url, method:'first' }, function(response) {
+	if(!response.status){
+			notify("URL not found in wayback archives!");
+		}	
 	});
 }
+function view_all_function(){
```
Comment (collinx) [inline scripts/popup.js]: @rgpt you have done a great job with the visual response. There is just this small glitch which I would like to point out. You have unintendedly created a duplicate function of `view_all_function()`.

**Comment by SerChirag** [comment]
Hi !
Clever of you, @rgpt to have replaced the annoying alert message with a cleaner and more subtle Notifications.
Now lets take this up a notch, shall we?
Why not disable the recent and first version button itself when we reach a page that has not been archived. Or simply just make their display property as "none".
You can get whether the page has been archived from this PR : https://github.com/internetarchive/wayback-machine-chrome/pull/35

Overall I think it would enhance user experience.
How do you feel about it ?
Any views are welcome :)

**Comment by rgpt** [comment]
Thank You @SerChirag  , i think anyhow you could enable the button by **display:block** from inspected elements if you see from developers point of view. Anyways we only want our user to get know whether this webpage is archived or not. we cannot hide everything if we do not have that archive not the social sharing button and similarly this.

in this #PR i just wanted to point out that there should be a response back to extension from content script which works with dom  and we should not use **alert** there notification can be of anytype but programatically it should work properly. 

Lets see what reviewers and others have to say on this ?

**Comment by rgpt** [comment]
@markjgraham @rchrd2  Kindly review this PR :)

**Pull request merged**